### DISCUSSION TITLE
网页端底部点击无法调用VLC和potplayer等播放器，点击之后无反应，软件都是正确安装好的，求解 需要如何操作

### DISCUSSION THREAD

**Discussion by Silentkiller47** [AlistGo/alist#3356] [Q&A]
网页端底部点击无法调用VLC和potplayer等播放器，点击之后无反应，软件都是正确安装好的，求解 需要如何操作
![alist](https://user-images.githubusercontent.com/41658969/218261960-d1f498a0-9cdf-41db-9aa6-b00c030610ae.png)

**Comment by liukunda** [comment]
同样没反应 复制链接也不好用

**Reply by haha-sky** [reply]
windows VLC调用需要安装一个组件，注册之后即可正常调用。[
参考：页面播放调用win本地vlc1：](https://zhihuizh.blogspot.com/2022/12/vlc-player.html)
[2](https://telegra.ph/alist-%E6%9C%AC%E5%9C%B0vlc%E8%B0%83%E7%94%A8-02-16)

**Reply by Deepseaon** [reply]
复制链接要去掉前缀，只能有http(s);//xxxxx这样链接才能用

**Reply by Dodfaf** [reply]
解决了吗
